dpgen2/exploration/report/report_trust_levels_spin.py

```
@@ -0,0 +1,300 @@
+        set_mf_fail = set(id_mf_fail)
+        # accu, cand, fail
+        set_accu = set_af_accu & set_mf_accu
+        set_cand = (
+            (set_af_cand & set_mf_accu)
+            | (set_af_cand & set_mf_cand)
+            | (set_af_accu & set_mf_cand)
+        )
+        set_fail = set_af_fail | set_mf_fail
+        # check size
+        assert nframes == len(set_accu | set_cand | set_fail)
+        assert 0 == len(set_accu & set_cand)
```

Ruff

Yoda condition detected

Rewrite as `len(set_accu & set_cand) == 0`

(SIM300)